src/btree/bt_io.c

```diff
@@ -365,6 +366,12 @@ __wt_bt_write(WT_SESSION_IMPL *session, WT_ITEM *buf,
 	WT_STAT_FAST_DATA_INCR(session, cache_write);
 	WT_STAT_FAST_CONN_INCRV(session, cache_bytes_write, dsk->mem_size);
 	WT_STAT_FAST_DATA_INCRV(session, cache_bytes_write, dsk->mem_size);
+	if (checkpoint_io) {
```

Comment:
I thought you only wanted the relationship between bytes written for checkpoint vs. bytes written for other purposes. How does having the absolute bytes written to disk instead of bytes pushed out of cache improve that information?

Regardless, we can know the disk-block write size in `bt_io.c` (and in fact, we have to figure it out there, for compressed and encrypted blocks), that's what the `bm->write_size()` call does.